velox/exec/tests/WindowTest.cpp

```diff
@@ -439,5 +439,46 @@ TEST_F(WindowTest, duplicateOrOverlappingKeys) {
       "Sorting keys must be unique and not overlap with partitioning keys. Found duplicate key: b");
 }
 
+TEST_F(WindowTest, memoryLeak) {
```

Comment:
```
s/memoryLeak/nagativeFrameArg/
```

Can we cover negative start/end and both with a for loop in this test case? Thanks!